uui/components/widgets/ProgressBar.tsx

```diff
@@ -13,7 +13,10 @@ export function applyProgressBarMods(mods: ProgressBarMods) {
     const size = mods.size || defaultSize;
 
     return [
-        css.root, css[`size-${size}`], mods.striped && css.striped,
+        'uui-progress_bar',
+        css.root,
+        css[`size-${size}`],
+        mods.striped && css.striped,
```

Comment:
```suggestion
        css.root,
        css[`size-${size}`],
        mods.striped && css.striped,
```
'uui-progress_bar' added to uui-components, look at snapshot

Reply:
fixed